doc/content/verification/ver-list.md

```diff
@@ -1,3 +1,9 @@
+For [SQA](sqa/index.md) purposes, TMAP8 undergoes verification and validation. Verification consists of comparing TMAP8 predictions against analytical solutions in different conditions, which are often simple cases. Validation consists of comparing TMAP8 predictions against experimental data.
```

Comment:
```suggestion
For [software quality assurance](sqa/index.md) purposes, TMAP8 undergoes verification and validation. Verification consists of comparing TMAP8 predictions against analytical solutions in different conditions, which are often simple cases. Validation consists of comparing TMAP8 predictions against experimental data.
```

We haven't defined this, and only use it once, so let's define it instead of using any acronyms.